dags/oup/parser.py

```diff
@@ -56,6 +57,15 @@ def __init__(self) -> None:
                 source="front/article-meta/title-group/article-title",
                 all_content_between_tags=True,
             ),
+            CustomExtractor(
+                destination="journal_volume",
+                extraction_function=self._get_journal_volume,
+            ),
+            CustomExtractor(
+                destination="journal_year",
+                extraction_function=self._get_journal_year,
+                required=True,
```

Comment:
default is True ;)

Reply:
In custom default is False

Reply:
https://github.com/cern-sis/workflows/blob/2644d9c5b223f0329b1a109c51efb5b233752bac/dags/common/parsing/xml_extractors.py#L128